static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -681,15 +690,19 @@ function GroupDetailsContent({
       openMergedIssuesDrawer();
     } else if (currentTab === Tab.ACTIVITY) {
       openIssueActivityDrawer();
+    } else if (seerDrawer) {
+      openSeerDrawer();
```

Comment:
hmm this will only work if hasStreamlinedUI is true which might be okay

Reply:
oh good catch, that was not intended, fixed now